ansible-ipi-install/roles/installer/defaults/main.yml

```diff
@@ -2,3 +2,21 @@
 # defaults file for installer
 cache_enabled: false
 provision_cache_store: "{{ ansible_env.HOME }}/rhcos_image_cache/"
+
+# The containers registry port
+registry_port_container: 5000
+
+# The port exposed on the server for the registry
+registry_port: "{{ registry_port_container }}"
+
+# The following variables define the registry location
+registry_dir: /opt/registry
+
+# When a registry is created, then information to connect 
+# to it is written to the following files.
+install_config_appends_file: install-config-appends.yml
+registry_auth_file: registry-auths.json
+
+# Default user and password for created registry
+disconnected_registry_user: dummy
+disconnected_registry_password: dummy
```

Comment:
is this user related to the remote user we use for the ansible install? i.e. kni?